sources/packages/web/src/components/common/notes/CreateNoteModal.vue

```diff
@@ -49,51 +73,52 @@ export default {
       required: true,
     },
   },
-  emits: ["submitData"],
-  setup(props: any, context: any) {
-    const { showDialog, showModal } = useModalDialog<void>();
-    const formData = ref();
-    const formioUtils = useFormioUtils();
-    const dialogClosed = () => {
-      showDialog.value = false;
-    };
-    const formLoaded = async (form: any) => {
-      formData.value = form;
-      const dropdown = formioUtils.getComponent(form, "noteType");
-      const options = [{}];
+
+  setup(props: any) {
+    const { showDialog, showModal, resolvePromise } = useModalDialog<
+      NoteAPIInDTO | boolean
+    >();
+    const addNewNoteForm = ref({} as VForm);
+    const formModel = reactive({} as NoteAPIInDTO);
+    const noteTypeItems = ref([] as NoteTypeItemsDTO[]);
+
+    onMounted(async () => {
       if (props.entityType === NoteEntityType.Institution) {
         for (const noteType in InstitutionNoteType) {
-          options.push({ label: noteType, value: noteType });
+          noteTypeItems.value.push({ title: noteType, value: noteType });
```

Comment:
Why are we using title and value here? We can still stick to the contract of option item like label and value right.